eng/pipelines/runtime-manual.yml

```diff
@@ -488,16 +488,16 @@ jobs:
       timeoutInMinutes: 180
       condition: >-
         or(
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_mono.containsChange'], true),
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_installer.containsChange'], true),
+          eq(variables['isManualOrIsNotPR'], true),
```

Comment:
The same  conditions you removed are duplicated in many other places of this file. Should we remove them too ?

Reply:
@steveisok would you prefer that? Or should I restore the conditions, and just add the `manual` one?

Reply:
I think that's ok.